Documentation/ApiOverview/Database/QueryBuilder/Index.rst

```diff
@@ -412,26 +412,25 @@ Create an :sql:`INSERT` query. Typical usage:
 Read :ref:`how to correctly instantiate <database-query-builder-instantiation>`
 a query builder with the connection pool.
 
-The `uid` of the created database row can be fetched from the connection
-by using :ref:`$queryBuilder->getConnection()->lastInsertId() <database-connection-last-insert-id>`.
-
 Remarks:
 
-*   Often it is often to use :php:`->insert()` or :php:`->bulkInsert()` of the
-    :ref:`Connection <database-connection>` object.
-
-*   :php:`->values()` expects an array of key/value pairs. Both keys (field
-    names / identifiers) and values are automatically quoted. In rare cases,
-    quoting of values can be turned off by setting the second argument to
-    :php:`false`. In those cases, quoting has must done manually, typically
-    by using :php:`->createNamedParameter()` on the values, use with care ...
+*   The `uid` of the created database row can be fetched from the connection
+    by using :ref:`$queryBuilder->getConnection()->lastInsertId() <database-connection-last-insert-id>`.
 
 *   :php:`->executeStatement()` after :php:`->insert()` returns the number of
     inserted rows, which is typically `1`.
 
-*   The query builder does not contain a method for inserting multiple rows at
-    once, use :php:`->bulkInsert()` of the :ref:`Connection
-    <database-connection>` object instead to achieve that.
+*   Often it might be more practical to use :php:`->insert()` of the
```

Comment:
Hmmm, I don't like the sentence: "be more practical" -> what does that mean?

And: I don't get it: "It is more practical ... because values() expects an array of key value pairs"